package.json

```diff
@@ -11,13 +11,17 @@
     "build": "parcel build src/index.html --out-dir build/release --public-url ./",
     "test": "jest -coverage",
     "lint": "tslint -p .",
-    "generate:definitions": "shx rm -rf ./generated/definitions && gen-api-models --client --request-types --response-decoders --api-spec ./src/generated/yaml/index.yaml --out-dir ./generated/definitions",
+    "generate:definitions": "shx rm -rf ./generated/definitions && gen-api-models --client --request-types --response-decoders --api-spec https://raw.githubusercontent.com/pagopa/io-functions-backoffice/master/openapi/index.yaml --out-dir ./generated/definitions",
     "generate": "npm-run-all generate:*",
     "prettify": "prettier --write \"./**/*.{ts,tsx}\""
   },
   "dependencies": {
     "bootstrap": "^4.5.3",
+    "fp-ts": "1.17.4",
+    "io-ts": "1.8.5",
+    "moment": "^2.29.1",
     "msal": "^1.4.4",
+    "react-moment": "^1.0.0",
```

Comment:
I suggest to use date-fns instead of moment which is going to be deprecated, see
https://momentjs.com/docs/#/-project-status/